tools/windows_buildenv.bat

```diff
@@ -29,8 +29,8 @@ IF DEFINED BUILDENV_RELEASE (
 ) ELSE (
     SET BUILDENV_BRANCH=2.6
     SET VCPKG_TARGET_TRIPLET=x64-windows
-    SET BUILDENV_NAME=mixxx-deps-2.6-x64-windows-12239ed
-    SET BUILDENV_SHA256=5c60b2c61d6448a99979d7cc997e92f2fd5b4b65f65e2439abfca3fa6fd30f8d
+    SET BUILDENV_NAME=mixxx-deps-2.6-x64-windows-7cc6152
+    SET BUILDENV_SHA256=44b0fc493959ffec09dc48af3b4400d3e550aed970187222f95558942a42e066
```

Comment:
In CI we use the release only version of the buildenv. The full version with additional debug binaries is for local development. This change will not enable ffmpeg therfore.

Reply:
Yes, I've commented that part out in the CI so I can start trying and see if I can get ffmpeg to work before submitting a PR to make a new vcpkg rel version